Dockerfile.k8sdataservice

```diff
@@ -0,0 +1,18 @@
+FROM python:3.10-slim-bookworm
+
+MAINTAINER shuliang@linkedin.com
+LABEL org.opencontainers.image.source=https://github.com/flyteorg/flytekit
+
+ARG CODE_DIR=/home/jobuser/code
+COPY plugins/flytekit-k8sdataservice $CODE_DIR/flytekit-k8sdataservice
```

Comment:
could we just install this plugin in this [dockerfile](https://github.com/flyteorg/flytekit/blob/23beed9d9e484eaa88c24a9d01a6638a3e7525b5/Dockerfile.agent#L30)?